packages/docs/content/04-clusters/1-new-clusters/05-gcp.md

```diff
@@ -148,16 +148,19 @@ The following steps need to be performed to provision a new GCP cluster:
    	 - Control plane Subnet
    	 - Worker Network
   * Make the choice of updating the worker pool in parallel.
-* Configure the master and worker node pools. A master and a worker node pool are configured by default.
-    - Name - A descriptive name for the node pool.
-    - Size - Number of VMs to be provisioned for the node pool. For the master pool, this number can be 1, 3, or 5.
-    - Allow worker capability (master pool) - Select this option for allowing workloads to be provisioned on master nodes.
-    - Instance type - Select the instance type to be used for all nodes in the node pool.
-    - Make your selection of Rolling Update of nodes. There are two choices of Rolling Update:
-		- Expand First: Launches the new node and then shut down the old node
-		- Contract First: Shut down the old node first and then launches the new node
-    - Availability Zones - Choose one or more availability zones. Spectro Cloud provides fault tolerance to guard against failures like hardware failures, network failures, etc. by provisioning nodes across availability zones if multiple zones are selected.
-    - Disk size - set the desired disk size.
+* Configure the master and worker node pools. A master and a worker node pool are configured by default.
+
+|Parameter| Description|
+|---------|---------------|   
+|Name |A descriptive name for the node pool|
+|Size | Number of VMs to be provisioned for the node pool. For the master pool, this number can be 1, 3, or 5|
+|Allow worker capability (master pool)|Select this option for allowing workloads to be provisioned on master nodes|
+|Instance type|Select the instance type to be used for all nodes in the node pool|
+|**Rolling Update**|There are two choices of Rolling Update|
+|Expand First|Launches the new node and then shut down the old node|
```

Comment:
it had (a) and (b) before, so either no a/b or has it everywhere for consistency please.